Insufficient types export (TS4023) of `Metadata`

> I am getting the same error for a different type (`Metadata`):

```
error TS4023: Exported variable '[...]' has or is using name Metadata from external module '[...]node_modules/.pnpm/express-zod-api@14.0.0_@types+express@4.17.20_@types+http-errors@2.0.3_@types+node@20.8.7_exp_ul3mzpgqwyhnfja3iatztge55q/node_modules/express-zod-api/dist/index' but cannot be named.
```

_Originally posted by @alindsay55661 in https://github.com/RobinTail/express-zod-api/issues/1182#issuecomment-1771435428_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,11 @@
 
 ## Version 14
 
+### v14.0.3
+
+- Fixed issue #1269 reported by [@alindsay55661](https://github.com/alindsay55661):
+  - `TS4023: Exported variable ... has or is using name Metadata from external module ... but cannot be named.`
+
 ### v14.0.2
 
 - Refactoring: consistent implementation for creating and starting HTTP and HTTPS servers in `createServer()` method.

--- a/example/example.swagger.yaml
+++ b/example/example.swagger.yaml
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Example API
-  version: 14.0.2
+  version: 14.0.3-beta1
 paths:
   /v1/user/retrieve:
     get:

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "express-zod-api",
-  "version": "14.0.2",
+  "version": "14.0.3-beta1",
   "description": "A Typescript library to help you get an API server up and running with I/O schema validation and custom middlewares in minutes.",
   "license": "MIT",
   "scripts": {

--- a/src/index.ts
+++ b/src/index.ts
@@ -37,7 +37,8 @@ export { Integration } from "./integration";
 
 export * as ez from "./proprietary-schemas";
 
-// Issues 952 and 1182: Insufficient exports for consumer's declaration
+// Issues 952, 1182, 1269: Insufficient exports for consumer's declaration
+export type { Metadata } from "./metadata";
 export type { ZodDateInDef } from "./date-in-schema";
 export type { ZodDateOutDef } from "./date-out-schema";
 export type { ZodFileDef } from "./file-schema";

--- a/src/metadata.ts
+++ b/src/metadata.ts
@@ -2,7 +2,7 @@ import { combinations } from "./common-helpers";
 import { z } from "zod";
 import { clone, mergeDeepRight } from "ramda";
 
-interface Metadata<T extends z.ZodTypeAny> {
+export interface Metadata<T extends z.ZodTypeAny> {
   examples: z.input<T>[];
 }
 

